### ISSUE TITLE
Active Element after successful update operation

### ISSUE THREAD

**Issue by hiteshah185** [TeraServ/starfish#81]
Keyboard Focus should be placed on the **edit icon** of the list/table page after **Successful update** operation.

**Comment by hiteshah185** [comment]
Can we use default focus restoration in MatDialog Config ?
```typescript
const dialogRef = this.dialog.open(DialogFromMenuExampleDialog, {restoreFocus: false});
```
